internal/storage/sql/common/rollout.go

```diff
@@ -567,20 +719,17 @@ func (s *Store) DeleteRollout(ctx context.Context, r *flipt.DeleteRolloutRequest
 		var rolloutID string
 
 		if err := rows.Scan(&rolloutID); err != nil {
-			_ = tx.Rollback()
 			return err
 		}
 
 		rolloutIDs = append(rolloutIDs, rolloutID)
 	}
 
 	if err := rows.Err(); err != nil {
-		_ = tx.Rollback()
 		return err
```

Comment:
does this set the named return `err` to the value of this `err` which would then trigger the `defer` functionality of performing the rollback?

I've always done something like:

```go
if cerr := rows.Err(); err != nil {
  err = cerr
  return
}
```

just to be sure